EmotiBit.cpp

```
@@ -4679,7 +4803,15 @@ void EmotiBit::printEmotiBitInfo()
     Serial.print(ip);
     Serial.println("\"");
   }
-  
+
+  if (!_bluetoothEnabled )
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)